README.md

```diff
@@ -49,6 +50,26 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### Mac OS
+
+ddev Cypress Setup (Mac)
+
+```bash
+# Install XQuartz
+brew install xquartz --cask
+
+# Run XQuartz
+open -a Xquartz
+```
+checkbox checkedIn the XQuartz preferences, go to the “Security” tab and make sure you’ve got the “Allow connections from network clients” ticked:
```

Comment:
I'm not sure what this is trying to say. Is it trying to say "Check the checkbox?" Which checkbox?

Reply:
I documented this clearly, including a screenshot, at https://github.com/TravisCarden/ddev-drupal-xb-dev/blob/v0.0.7/README.md#xquartz.